### PR TITLE
Updated spawner docs

### LINKED ISSUE #822: Remove "Writing a custom spawner" heading
In http://jupyterhub.readthedocs.io/en/latest/spawners.html#writing-a-custom-spawner, the heading exists but there is no content. Please remove the heading.

### PR DESCRIPTION
**Pull request by Scrypy**
I updated the spawner docs include a link to the tutorial.
closes #822 

I worked with @captainsafia on this at PyData Chicago.

### CODE REVIEW

File: docs/source/spawners.md [hunk 1]
```diff
@@ -163,4 +163,6 @@ When `Spawner.start` is called, this dictionary is accessible as `self.user_opti
 [Spawner]: https://github.com/jupyter/jupyterhub/blob/master/jupyterhub/spawner.py
 
 
-## Writing a custom spawner
\ No newline at end of file
+## Writing a custom spawner
+
+If you are interested in building a custom spawner, you can read [this tutorial](http://jupyterhub-tutorial.readthedocs.io/en/latest/spawners.html)
```
Comment (willingc) [inline docs/source/spawners.md]: Hi @Scrypy. Thanks for the contribution. One small change would be to add a period at the end of the sentence. Thanks!

Comment (Scrypy) [inline docs/source/spawners.md]: Hi @willingc , sorry, trying to fix it now. I'm having problems with git because I'm trying to do another commit, push and then `git remote add upstream https://github.com/jupyterhub/jupyterhub.git` and I get the error `remote upstream already exists.`.
Should I just try with a different name other than upstream?

Comment (willingc) [inline docs/source/spawners.md]: Hi @scrypy, I'm going to merge as is and then you can open a new PR to take care of the period at the end of the sentence. Yay!

Thanks and you can check your remote settings with `git remote -v`. 

You may find this talk on git that I did in the past helpful: http://www.slideshare.net/willingc/yes-you-can-git

**Comment by willingc** [comment]
Thanks :tada:

File: docs/source/spawners.md (hunk 1 above)
Comment (Scrypy) [inline docs/source/spawners.md]: Thanks for the link! I'll check it out and I'll to fix this and more issues! 😬

Comment (captainsafia) [inline docs/source/spawners.md]: Daniel — let me know if the link is helpful. If not, be sure to attend the next PyData open source night and I can help you with it.
